contracts/modules/LidoWrapHandler.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import '@yield-protocol/utils-v2/contracts/token/IERC20.sol';
+import '@yield-protocol/utils-v2/contracts/token/TransferHelper.sol';
+import '../LadleStorage.sol';
+import '../oracles/lido/IWstETH.sol';
+
+/// @dev A contract to handle wrapping & unwrapping of stETH
+contract LidoWrapHandler {
+    using TransferHelper for IERC20;
+    using TransferHelper for IWstETH;
```

Comment:
Does this work? I think that `TransferHelper.safeTransfer()` takes an IERC20 as the first parameter.

When using `using`, the function can be appended to a variable with the same type as its first parameter.

Below, you can cast wstETH into an IERC20 by doing `IERC20(address(wstETH))`

Reply:
This is working in the local environment. IWstETH is inheriting from IERC20 that might be the reason for this to work?

Reply:
Ah, that's the reason, cool!